pyproject.toml

```diff
@@ -93,6 +94,7 @@ dev = [
     "grayskull",
     "build",
     "jinja2",
+    "setuptools==63.4.3",
```

Comment:
Is there conflict/problem to upgrade this to latest (`68.2.2`)?

Reply:
This seems to be an issue with Keycloak: https://github.com/marcospereirampj/python-keycloak/pull/483

Being tracked here: https://github.com/nebari-dev/nebari/issues/1984